soh/soh/Enhancements/gameplaystats.h

```diff
@@ -87,7 +87,7 @@ typedef enum {
     COUNT_ENEMIES_DEFEATED_PARASITIC_TENTACLE,   // EN_BA
     COUNT_ENEMIES_DEFEATED_PEAHAT,               // EN_PEEHAT
     COUNT_ENEMIES_DEFEATED_PEAHAT_LARVA,         // EN_PEEHAT
-    COUNT_ENEMIES_DEFEATED_POE,                  // EN_POH
+    COUNT_ENEMIES_DEFEATED_POE,                  // EN_POH & EN_PO_FIELD
```

Comment:
is this bringing things in line with how it was actually working before or changing the meaning?

was it

`COUNT_ENEMIES_DEFEATED_POE` means "non-big poes defeated" and  `COUNT_ENEMIES_DEFEATED_POE_BIG` means "big poes defeated" before, and this is changing it to mean `COUNT_ENEMIES_DEFEATED_POE` meaning "big and non-big poes defeated" and `COUNT_ENEMIES_DEFEATED_POE_BIG` meaning "big poes defeated"

or was it always

`COUNT_ENEMIES_DEFEATED_POE` meaning "big and non-big poes defeated" and `COUNT_ENEMIES_DEFEATED_POE_BIG` meaning "big poes defeated"

Reply:
`COUNT_ENEMIES_DEFEATED_POE` means "non-big poes defeated" but `ACTOR_EN_PO_FIELD` is used by every poe in hyrule field, even the small ones, I just updated the comment to reflect that.

Here's the code for counting the kills:
```cpp
case ACTOR_EN_POH:
    if (actor->params == EN_POH_FLAT || actor->params == EN_POH_SHARP) {
        gSaveContext.sohStats.count[COUNT_ENEMIES_DEFEATED_POE_COMPOSER]++;
    } else {
        gSaveContext.sohStats.count[COUNT_ENEMIES_DEFEATED_POE]++;
    }
    break;

case ACTOR_EN_PO_FIELD:
    if (actor->params == EN_PO_FIELD_BIG) {
        gSaveContext.sohStats.count[COUNT_ENEMIES_DEFEATED_POE_BIG]++;
    } else {
        gSaveContext.sohStats.count[COUNT_ENEMIES_DEFEATED_POE]++;
    }
    break;
```